Source/Core/Core/HW/GCMemcard/GCMemcard.cpp

```diff
@@ -298,25 +301,25 @@ u32 GCMemcard::TestChecksums() const
 
   u32 results = 0;
 
-  calc_checksumsBE((u16*)&m_header_block, 0xFE, &csum, &csum_inv);
+  std::tie(csum, csum_inv) = m_header_block.CalculateChecksums();
```

Comment:
Now that we're officially C++17, you could get back to using structured bindings ;)

Reply:
Haha, sure, why not.

Reply:
Alright, done.